cms/static/cms/js/modules/cms.base.js

```diff
@@ -39,6 +39,8 @@ var CMS = {
     $(document).ready(function () {
         CMS.API.Helpers = {
 
+            isTouch: window.ontouchstart !== undefined,
```

Comment:
well, the events are normalized with pep.js, so showTooltip comes from `pointerover`. There is a `originalEvent.pointerType` property, but it gets `mouse` on ipad, so it's not robust at all.

I believe we'll need a solid solution to define whether we are dealing with a touch device anyway, so I'll look into better wayto implement touch detection. Modernizr does it somehow